data/json/items/tools.json

```diff
@@ -966,21 +966,6 @@
       "skill2": "computer"
     }
   },
-  {
-    "id": "brazier",
-    "type": "TOOL",
-    "name": "brazier",
-    "description": "This is a large metal stand with slots on the side.  You can set it somewhere through the construction menu and then set fires in it with no risk of spreading.",
-    "weight": 2000,
-    "volume": 6,
-    "price": 9000,
-    "to_hit": 1,
-    "bashing": 7,
-    "material": "steel",
-    "symbol": ";",
-    "color": "light_gray",
-    "flags": [ "TRADER_AVOID" ]
```

Comment:
Still see some "hidden" rebalance in weight \ volume \ price, etc. It's ok for me but it's better to explicitly state that.

Also, you didn't copy the flag, I guess it was here intentionally so it's better to keep it (if you can't explain its avoidance).

Reply:
I left the new stats in the deployable.json brazier because I think they're more realistic for a mid-sized steel brazier. TRADER_AVOID was removed because the brazier is a fairly typical item with nothing that should cause undesirable behavior with NPCs.

Reply:
What about removed bashing property?
Now it significantly differs from portable kiln and wooden log. It's a big thing that can hurt, isn't?